### PR TITLE
make stddev work

### PR DESCRIPTION
**Pull request by neonstalwart**
it looks like stddev isn't working but there are no tests covering it

### CODE REVIEW

**Comment by neonstalwart** [comment]
i expect this PR will fail as it is (i have an extra commit that should fix it) but this seems like a test script failure

cc @otoolep @corylanou

**Comment by toddboom** [comment]
@neonstalwart agreed, that's a weird one. kicking the build to see if it pulls down correctly the next time.

**Comment by neonstalwart** [comment]
@toddboom :disappointed: didn't work

**Comment by otoolep** [comment]
Build script has been fixed, please rebase so CI testing can re-run.

**Comment by neonstalwart** [comment]
i need some help here... i'm fairly sure that `ReduceStddev` is getting something like `[[1,2,3],[4,5,6]]` (i.e. `[][]float64`) but i don't understand go well enough to come up with the right syntax to make this work.  `ReduceStddev` seems to be the only function that doesn't get just a flat slice of values.  any advice?

**Comment by neonstalwart** [comment]
hmm... i've got this working for TestSingleServer but it fails for Test3NodeServer.  anyone got an idea?

**Comment by neonstalwart** [comment]
i think i've actually got this PR working.  can someone take a look at why this failed?  i think it's a CI failure rather than a real failure but i don't understand what's happened.

cc @otoolep @toddboom

**Comment by otoolep** [comment]
Thanks @neonstalwart -- it is a failure unrelated to your change. We are currently investigating the root cause of these failures in CI.

**Comment by neonstalwart** [comment]
roll the :game_die: again and this time it passes :tada:

**Comment by neonstalwart** [comment]
this is ready for review now - once the values were properly unmarshalled everything started to make more sense.

**Comment by neonstalwart** [comment]
i'm not sure who is responsible for which parts of influxdb but it looks like @pauldix shows up a lot in the blame for influxql/functions.go so he gets a mention to see if this could get a review.

File: influxql/functions.go [hunk 1]
```diff
@@ -419,7 +425,7 @@ func ReduceStddev(values []interface{}) interface{} {
 		sq := math.Pow(dif, 2)
 		variance += sq
 	}
-	variance = variance / float64(len(data)-1)
+	variance = variance / float64(count-1)
```
Comment (pauldix) [inline influxql/functions.go]: Should check for count == 1 or this'll be bad

Comment (neonstalwart) [inline influxql/functions.go]: look at line 410

Comment (neonstalwart) [inline influxql/functions.go]: it probably wouldn't be a bad idea to add a test for `count < 2` though.  i can work on that if you want.

Comment (pauldix) [inline influxql/functions.go]: ah,  no, that's fine, didn't see it.

**Comment by corylanou** [comment]
+1

**Comment by neonstalwart** [comment]
i went ahead and added a test for one point - i think it's a significant case.

**Comment by pauldix** [comment]
Thanks @neonstalwart!

**Pull request merged**